R/Archiver.R

```diff
@@ -78,15 +81,16 @@ JSONArchiver <- R6::R6Class( # nolint: object_name_linter.
     #'
     #' @param reporter `Reporter` instance.
     #'
-    #' @return invisibly self
+    #' @return `self`, invisibly.
     #' @examples
+    #' library(ggplot2)
```

Comment:
`ggplot2` is a `Suggests` dependency, please use `@examplesIf` with `requireNamespace`.